stable/newrelic-infrastructure/values.yaml

```diff
@@ -0,0 +1,59 @@
+licenseKey: undefined
+
+verboseLog: false
+
+# This can be set, the default is shown below
+# logFile: /var/log/nr-infra.log
+
+image:
+  name: newrelic/infrastructure
+  tag: 0.0.7
+  pullPolicy: IfNotPresent
+
+resources:
```

Comment:
We usually recommend not to specify defaults for resources leaving this a conscious choice of the user.

Reply:
👍

Reply:
I'm not sure I agree, if the user knows little about the application then they might not know what is a sane default. In this case these defaults come from the vendor of the application, so can be trusted (imho).  I looked through all the stable charts before making this choice 52 specify resources with only 18 leaving them empty. 

Happy to remove if its a requirement of entry but I wanted to explain my thinking here!

Reply:
The reason we now tend to recommend to not specify defaults is that users should be able to install/test charts on small systems, such as Minikube. Oftentimes, defaults were kind of really high. In your case, I guess you can leave them in.